bazel/dependency_imports.bzl

```diff
@@ -11,6 +11,28 @@ load("@protodoc_pip3//:requirements.bzl", protodoc_pip_install = "pip_install")
 # go version for rules_go
 GO_VERSION = "1.14.4"
 
+def envoy_api_dependency_import():
```

Comment:
How come this change is made to the Envoy `bazel/` rather than `api/bazel/`? If this is something needed for any use of the APIs, it probably belongs in the latter.